### ISSUE TITLE
AuthLink Helper does not appear to work in view cells.

### ISSUE THREAD

**Issue by alexlambson** [CakeDC/users#465]
Right now I am trying to call AuthLink helper in a view cell's ctp and nothing is appearing. 

I have permission because I can view the page by going to the URL. I know authlink is working on the page because I use it twice in the layout and once in the template. 

The only place I can't get it to work is in view cell templates. 

viewcell.ctp
```
<div class="edit-delete">
    <?= $this->AuthLink->link(
                '', 
                ['controller' => 'events', 'action' => 'ShowEventRegistration', $event->id], 
                ['class' => 'edit']
            ); 
    ?>
</div>
```

AppView.php

```
public function initialize(){
    $this->loadHelper('CakeDC/Users.User');
    $this->loadHelper('CakeDC/Users.AuthLink');
}
```

**Comment by steinkel** [comment]
closed per #485

**Comment by faiyazalam** [comment]
I have same issue in CakeDC Users 5.0 
https://github.com/CakeDC/users/issues/538
